draft-pauly-oblivious-doh.md

```diff
@@ -217,6 +217,15 @@ content-length = 154
 <Bytes containing the encrypted payload for an Oblivious DNS response>
 ~~~
 
+The following example shows a 4xx (Client Error) response that can be sent from an Oblivious
```

Comment:
I'm not sure we need an example here--if anything, it is getting us into murkier waters, since an error can have a body that explains the error. How about we just explain that you should send a 4xx error if the request cannot be processed?

Reply:
Oh, fair. I'll just replace it with text.

Reply:
Should be fixed now!